src/components/page/ohri-form-page.scss

```diff
@@ -40,15 +40,14 @@
   margin-bottom: 0 !important;
 }
 
-.embeddedFormSection{
+.embeddedFormSection {
   background-color: colors.$gray-10;
   margin-left: 0.75rem;
   padding: 0.5rem 0 0.5rem 0.75rem;
-  
 }
 
 .collapseToggle {
-  margin-Left: -180px;
+  margin-left: -180px;
```

Comment:
Nice catch @denniskigen! I recently noticed that the styling of the collapse toggle was broken, I'm curious on how this rule impacts the positioning of the toggle.